.travis.yml

```diff
@@ -54,6 +54,12 @@ matrix:
     env: STYLE_CHECK=true
     script: ./sbt scalastyle
 
+  - scala: 2.11.2
+    script:
```

Comment:
It's minor, but for convenience and consistency, could you please add `env` section to this build with something descriptive and human-readable, like `FE_PYTHON_TEST=true` ?

It does not mean much yet, but is super-helpful with TravisCI, when a project has many profiles, to quickly identify the purpose of the failing one. 

Like i.e [here](https://travis-ci.org/src-d/gemini/builds/332875688) one would be able to tell that it's only Integration Tests failing for this job.
<img width="1048" alt="screen shot 2018-03-21 at 3 05 06 pm" src="https://user-images.githubusercontent.com/5582506/37697706-4ad8e58a-2d19-11e8-9818-75f991dae821.png">